tests/v3/test_store/test_memory.py

```diff
@@ -38,3 +38,36 @@ def test_store_supports_partial_writes(self, store: MemoryStore) -> None:
 
     def test_list_prefix(self, store: MemoryStore) -> None:
         assert True
+
+
+class TestGpuMemoryStore(StoreTests[GpuMemoryStore]):
+    store_cls = GpuMemoryStore
+
+    def set(self, store: GpuMemoryStore, key: str, value: Buffer) -> None:
+        store._store_dict[key] = value
+
+    def get(self, store: MemoryStore, key: str) -> Buffer:
+        return store._store_dict[key]
+
+    @pytest.fixture(scope="function", params=[None, {}])
+    def store_kwargs(self, request) -> dict[str, str | None | dict[str, Buffer]]:
+        return {"store_dict": request.param, "mode": "w"}
+
+    @pytest.fixture(scope="function")
+    def store(self, store_kwargs: str | None | dict[str, GpuBuffer]) -> GpuMemoryStore:
+        return self.store_cls(**store_kwargs)
+
+    def test_store_repr(self, store: GpuMemoryStore) -> None:
+        assert str(store) == f"gpumemory://{id(store._store_dict)}"
+
+    def test_store_supports_writes(self, store: GpuMemoryStore) -> None:
+        assert store.supports_writes
+
+    def test_store_supports_listing(self, store: GpuMemoryStore) -> None:
+        assert store.supports_listing
+
+    def test_store_supports_partial_writes(self, store: GpuMemoryStore) -> None:
+        assert store.supports_partial_writes
+
+    def test_list_prefix(self, store: GpuMemoryStore) -> None:
+        assert True
```

Comment:
Is there something to actually test here?

Reply:
AFAIU, this is here just to silence the xfail in the base `StoreTests` class: https://github.com/zarr-developers/zarr-python/blob/v3/src/zarr/testing/store.py#L175